docs/vmanomaly.md

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
it is better to put images next to the vmanomaly markdown file to remain consistent with other docs in the folder.